src/main/java/com/iexec/common/replicate/ReplicateStatusDetails.java

```diff
@@ -27,25 +27,30 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class ReplicateStatusDetails {
-
-    public static final int MAX_STDOUT_LENGTH = 100000;
-
     private ChainReceipt chainReceipt;
     private String resultLink;
     private String chainCallbackData;
     private String errorMessage;
     private ReplicateStatusCause cause;
-    private String stdout;
+    private ComputeLogs computeLogs;
     private Integer exitCode; //null means unset
+    private String teeSessionGenerationError; // null means unset
 
     public ReplicateStatusDetails(ReplicateStatusDetails details) {
         chainReceipt = details.getChainReceipt();
         resultLink = details.getResultLink();
         chainCallbackData = details.getChainCallbackData();
         errorMessage = details.getErrorMessage();
         cause = details.getCause();
-        stdout = details.getStdout();
+        computeLogs = details.getComputeLogs() == null
```

Comment:
Can you explain why not
`computeLogs = details.getComputeLogs();`
directly?

Reply:
Because if we'd call `tailLogs` on either the `ReplicateStatusDetails` base or the copy, it would also tails on the other one. So duplicating there seems mandatory.